p2p_wallet/Scenes/CreateOrRestoreWallet/CreateWallet/CreateSecurityKeys/CreateSecurityKeys.NewRootView.swift

```diff
@@ -0,0 +1,115 @@
+//
+//  CreateSecurityKeys.RootView.swift
+//  p2p_wallet
+//
+//  Created by Chung Tran on 22/02/2021.
+//
+
+import UIKit
+import TagListView
+import RxSwift
+import RxCocoa
+import Action
+
+extension CreateSecurityKeys {
+    class NewRootView: ScrollableVStackRootView {
+        // MARK: - Dependencies
+        @Injected private var viewModel: CreateSecurityKeysViewModelType
+        
+        // MARK: - Properties
+        private let disposeBag = DisposeBag()
+        
+        // MARK: - Subviews
+        private let navigationBar: WLNavigationBar = {
+            let navigationBar = WLNavigationBar(forAutoLayout: ())
+            navigationBar.titleLabel.text = L10n.yourSecurityKey
+            return navigationBar
+        }()
+        
+        private let saveToICloudButton: WLStepButton = {
+            let attrString = NSMutableAttributedString()
```

Comment:
- The apple logo has been added, merge develop to get it, don't use apple symbol, because it's hard to modify vertical position with text.
- Don't override the attributedString, use default builder instead (example in RestoreWallet.ViewController), 
```swift
WLStepButton.main(image: .appleLogo, text: L10n.restoreUsingICloud)
```